### ISSUE TITLE
Elements of the GUI flashing

### ISSUE THREAD

**Issue by GregoryTravis** [enso-org/enso#7760]
### What type of issue is this?

Transient – Occurring only once

### Is this issue blocking you from using Enso?

- [ ] Yes, I can't use Enso because of this issue.

### Is this a regression?

- [ ] Yes, previous version of Enso did not have this issue.

### What issue are you facing?

After working for about 30 minutes, running a local build against the cloud backend, elements of the GUI started flashing. It became hard to click on, for example, the arrow to change the visualization type.

### Expected behaviour

I expect no flashing, and for the user interface elements to be easily clickable.

### How we can reproduce it?

I wasn't able to reproduce it. Restarting the IDE made the problem go away.

### Screenshots or screencasts

https://github.com/enso-org/enso/assets/24406074/bf58ca26-bc22-4cb4-936d-aa99dcb3483a

https://github.com/enso-org/enso/assets/24406074/8cefcb32-d04b-4b2f-b6e2-4bbe8adbce30

### Logs

[2023-09-06T19-27-53.858Z-ide-2023.2.1-dev.log](https://github.com/enso-org/enso/files/12542650/2023-09-06T19-27-53.858Z-ide-2023.2.1-dev.log)
[client-Wed-Sep-6-16.28.55-2023.txt](https://github.com/enso-org/enso/files/12542651/client-Wed-Sep-6-16.28.55-2023.txt)
[server-Wed-Sep-6-16.28.57-2023.txt](https://github.com/enso-org/enso/files/12542652/server-Wed-Sep-6-16.28.57-2023.txt)
[2023-09-06T20-30-16.627Z-ide-2023.2.1-dev.log](https://github.com/enso-org/enso/files/12542653/2023-09-06T20-30-16.627Z-ide-2023.2.1-dev.log)
[client-Wed-Sep-6-17.07.44-2023.txt](https://github.com/enso-org/enso/files/12542654/client-Wed-Sep-6-17.07.44-2023.txt)
[server-Wed-Sep-6-17.07.47-2023.txt](https://github.com/enso-org/enso/files/12542655/server-Wed-Sep-6-17.07.47-2023.txt)

### Enso Version

local build on branch wip/gmt/5105-post

### Browser or standalone distribution

Standalone distribution (cloud project)

### Browser Version or standalone distribution

standlone

### Operating System

MacOS

### Operating System Version

Monterey 12.6.3

### Hardware you are using

MacBook Air (M1, 2020)

**Comment by vitvakatu** [comment]
Please let me know if you're able to reproduce it again.  For now, we are not going to investigate it.

**Comment by kazcw** [comment]
Duplicate of #7026.
